internal/controller/common.go

```diff
@@ -284,3 +274,9 @@ func mapsSorted[K cmp.Ordered, V any](m map[K]V) iter.Seq2[K, V] {
 		}
 	}
 }
+
+// backoffDelayAt returns the backoff delay at the given retry count.
+// Backoff is exponential with base 5 seconds.
+func backoffDelayAt(retry int) time.Duration {
+	return time.Duration(retry) * 5 * time.Second
```

Comment:
What happen on the first failure?

Reply:
With the first failure, the `spec.failed` will be increased to 1. This logic occurs before the backoff duration calculation